source/elements/oneDPL/source/parallel_api/execution_policies.rst

```diff
@@ -180,5 +178,54 @@ as the template argument, otherwise unspecified.
 Return a policy object constructed from ``policy``, with a new kernel name provided as the template
 argument. If no policy object is provided, the new policy is constructed from ``dpcpp_default``.
 
+.. _exec-policy-type-trait:
+
+Execution Policy Type Trait
++++++++++++++++++++++++++++
+
+oneDPL provides type trait utilities to detect its execution policy types at compile time for the purpose of
+function overload resolution:
+
+.. code:: cpp
+
+  // Defined in <oneapi/dpl/execution>
+
+  namespace oneapi {
+    namespace dpl {
+
+      template <class T>
+      struct is_execution_policy { /*see below*/ };
+
+      template <class T>
+      constexpr bool is_execution_policy_v = oneapi::dpl::is_execution_policy<T>::value;
```

Comment:
```suggestion
      inline constexpr bool is_execution_policy_v = oneapi::dpl::is_execution_policy<T>::value;
```

Reply:
It's a template, `inline` should not be needed. See the standard (https://eel.is/c++draft/execution.syn):
```
namespace std {
  // [[execpol.type]](https://eel.is/c++draft/execpol.type), execution policy type trait
  template<class T> struct is_execution_policy;
  template<class T> constexpr bool [is_execution_policy_v](https://eel.is/c++draft/execution.syn#lib:is_execution_policy_v) = is_execution_policy<T>::value;
}
```

Reply:
OK, seems `inline` is necessary for C++!7/20. Thanks, I will get it back.